src/controllers/api.controller.ts

```diff
@@ -1,11 +1,14 @@
 import { Request, Response, NextFunction } from "express";
 import { prisma } from "../db/prisma";
 import { Prisma } from "prisma/prisma-client";
-import { passToExpressErrorHandler, turnFalsyPropsToUndefined, poormansNormalizer } from "../utils";
+import {
+  passToExpressErrorHandler,
+  turnFalsyPropsToUndefined,
+  poormansNormalizer,
+} from "../utils";
 import createError from "http-errors";
 import { validateNoLeadingZeroes } from "ethereumjs-util";
 
-
 export const claimPost = async (
```

Comment:
in the future lets separate out the lint prs from the functional ones, for now its ok